### ISSUE TITLE
_get_sorting_curation has incorrect conditional test for action type

### ISSUE THREAD

**Issue by jsoules** [magland/sortingview#64]
In ~/src/python/sortingview/workspace/workspace.py current lines 328-351 (https://github.com/magland/sortingview/blob/b45d63d531186212ae8abcbb63043eed819ad8ae/src/python/sortingview/workspace/workspace.py#L328-L351) there are two versions of the test against action type. The first two checks compare the message type with the expected strings from the React-side reducer, but the second two checks a) incorrectly set a default result if the string is not found and b) don't actually check the value returned. As a result, any curation action other than adding or removing a unit label would be handled as a unit merge (with no ability to unmerge).

**Comment by jsoules** [comment]
Steps to reproduce:

* Open a sorting (having curation rights)
* Merge two or more units through the UI
* Unmerge them through the UI
* Attempt to load the workspace curation:
```python
>>> import sortingview as sv
>>> workspace_uri='workspace://[URI from the GUI]'
>>> W = sv.load_workspace(workspace_uri)
>>> W.get_sorting_curation('[sorting key from the GUI URL]')
```

Expected result should honor the unmerge actions (meaning you should not see your merged-and-unmerged group as still merged), but actual result will implement only merges and never unmerges, rapidly approaching a state where all units are merged down to a single one.

**Comment by magland** [comment]
Fixed by #65 and will be included in next package release.
